pkg/common/volume_operations.go

```diff
@@ -346,8 +346,9 @@ func (vo *VolumeOperationsImpl) deleteLVGIfVolumesNotExistOrUpdate(lvg *lvgcrd.L
 		"volumeID": volID,
 	})
 
+	driveUUID := append(vo.k8sClient.GetSystemDriveUUIDs(context.Background()), base.SystemDriveAsLocation)
```

Comment:
driveUUID it is a slice, let's use more meaningful name: `driveUUIDs`. When  read name like `driveUUID` I suppose that this variable stores something singular, like string.